transforms/images/polus-imagej-filter-frangivesselness-plugin/plugin.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "ImageJ filter frangiVesselness",
+  "version": "0.4.0",
+  "title": "ImageJ filter frangiVesselness",
+  "description": "Applies the Frangi Vesselness filter to an input collection to highlight vessel-like structures",
+  "author": "Nick Schaub (nick.schaub@nih.gov), Anjali Taneja, Benjamin Houghton (benjamin.houghton@axleinfo.com)",
+  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+  "repository": "https://github.com/labshare/polus-plugins",
+  "website": "https://ncats.nih.gov/preclinical/core/informatics",
+  "citation": "Frangi, A. F., Niessen, W. J., Vincken, K. L., & Viergever, M. A. (1998). Multiscale vessel enhancement filtering. In Medical Image Computing and Computer-Assisted Intervention — MICCAI'98 (pp. 130-137). Springer Berlin Heidelberg. doi:10.1007/bfb0056195",
+  "containerId": "polusai/imagej-filter-frangivesselness-plugin:0.4.0",
+  "inputs": [
+    {
+      "name": "opName",
+      "type": "enum",
+      "options": {
+        "values": ["DefaultFrangi"]
+      },
+      "description": "Operation to perform",
+      "required": true
+    },
+    {
+      "name": "inpDir",
+      "type": "collection",
+      "description": "Collection to be processed by this plugin",
+      "required": true
+    },
+    {
+      "name": "spacing",
+      "type": "array",
+      "description": "The physical distance between data points in the  image, can vary for each dimension of the image.",
+      "required": true,
+      "options":{
+        "items": {
+          "type": "string",
+          "title": "The physical distance between data points in the  image, can vary for each dimension of the image.",
+          "widget": "string"
+        }
+      }
+    },
+    {
+      "name": "scale",
+      "type": "number",
+      "description": "The input collection scale",
```

Comment:
This is actually the pixel size in distance units?

Reply:
This is the Frangi scale parameter, description updated in `main.py`, `plugin.json`, and `README.md`